plugins/mcreator-localization/lang/texts_fr_FR.properties

```diff
@@ -231,15 +231,15 @@ blockly.block.item_get_enhancement=Obtenir le level de l'enchantement de %1 de t
 blockly.block.deal_damage=Infligez %1 dégâts au type %2\:
 blockly.block.entity_nbt_text_set=Définir l'étiquette NBT de données de texte %3 à %1 à %2
 blockly.block.blockat_hardness=Obtenir la dureté du block à x\: %1 y\: %2 z\: %3
-blockly.block.gui_get_text_textfield=text inside textfield %1
+blockly.block.gui_get_text_textfield=texte dans le champ textuel %1
 blockly.block.entity_health=Vie actuelle de %1
 blockly.block.entity_size_height=Récupérer la hauteur de %1
 blockly.block.entity_size_width=Récupérer la largeur de %1
 blockly.block.entity_submerged_height=Get submerged height of %1
-blockly.block.gui_remove_items=Remove %1 items from slot %2 of the currently open GUI of %3
+blockly.block.gui_remove_items=Enlever %1 items du slot %2 du GUI actuellement ouvert pour %3
```

Comment:
Replace "items" by  "objets" and "slot" by "emplacement"

Reply:
These are automated pulls from crowdin. This should be changed on the Crowdin translation page and it will be synced here, otherwise it will be overriden again.